plugins/lookup/dsv.py

```diff
@@ -0,0 +1,123 @@
+# python 3 headers, required if submitting to Ansible
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+lookup: dsv
+author: Adam Migus (adam@migus.org)
+short_description: Get secrets from Thycotic DevOps Secrets Vault
+description:
+    - Uses the Thycotic DevOps Secrets Vault Python SDK to get Secrets from a
+       DSV `tenant` using a `client_id` and `client_secret`.
+requirements:
+    - python-dsv-sdk - https://pypi.org/project/python-dsv-sdk/
+options:
+    _terms:
+        description: the path to the secret e.g. /staging/servers/web1
+        required: True
+    tenant:
+        description: the first format parameter in the default `url_template`
+        env:
+            - name: DSV_TENANT
+        ini:
+            - section: dsv_lookup
+              key: tenant
+        required: True
+    tld:
+        default: com
+        description: the top-level domain of the tenant; the second format
+            parameter in the default `url_template`
+        env:
+            - name: DSV_TLD
+        ini:
+            - section: dsv_lookup
+              key: tld
+        required: False
+    client_id:
+        description: the client_id with which to request the Access Grant
+        env:
+            - name: DSV_CLIENT_ID
+        ini:
+            - section: dsv_lookup
+              key: client_id
+        required: True
+    client_secret:
+        description: the client_secret associated with the specific client_id
+        env:
+            - name: DSV_CLIENT_SECRET
+        ini:
+            - section: dsv_lookup
+              key: client_secret
+        required: True
+    url_template:
+        default: https://{}.secretsvaultcloud.{}/v1
+        description: the path to append to the base URL to form a valid REST
+            API request
+        env:
+            - name: DSV_URL_TEMPLATE
+        ini:
+            - section: dsv_lookup
+              key: url_template
+        required: False"""
+
+RETURN = r"""
+_list:
+    description:
+        - One or more JSON responses to `GET /secrets/{path}`
+        - See https://dsv.thycotic.com/api/index.html#operation/getSecret"""
+
+EXAMPLES = r"""
+- hosts: localhost
+  vars:
+      secret: "{{ lookup('dsv', '/test/secret') }}"
+  tasks:
+      - debug: msg="the password is {{ secret["data"]["password"] }}"
```

Comment:
Please use YAML syntax:
```suggestion
      - debug:
          msg: 'the password is {{ secret["data"]["password"] }}'
```